src/main/java/hudson/plugins/jacoco/JacocoBuildAction.java

```diff
@@ -201,12 +197,12 @@ public Object getTarget() {
 	}
 
 	@Override
-	public AbstractBuild<?,?> getBuild() {
+	public Run<?,?> getBuild() {
```

Comment:
Backward incompatibility here. You should use `@WithBridgeMethods`, [this](https://github.com/jenkinsci/analysis-core-plugin/blob/9f8f492d2931981f91a06a8eb0123fd7d3cd1a58/src/main/java/hudson/plugins/analysis/views/ConsoleDetail.java#L114-L128) is an example.